Riot/Modules/Room/RoomViewController.m

```diff
@@ -5181,5 +5188,25 @@ - (void)roomContextualMenuViewControllerDidReaction:(RoomContextualMenuViewContr
     [self hideContextualMenuAnimated:YES];
 }
 
+#pragma mark - ReactionsMenuViewModelCoordinatorDelegate
+
+- (void)reactionsMenuViewModel:(ReactionsMenuViewModel *)viewModel didAddReaction:(NSString *)reaction forEventId:(NSString *)eventId
+{
+    [self.roomDataSource addReaction:reaction forEventId:eventId success:^(NSString *eventId) {
+        
+    } failure:^(NSError *error) {
+        
+    }];
+}
+
+- (void)reactionsMenuViewModel:(ReactionsMenuViewModel *)viewModel didRemoveReaction:(NSString *)reaction forEventId:(NSString *)eventId
+{
+    [self.roomDataSource removeReaction:reaction forEventId:eventId success:^{
+        
+    } failure:^(NSError *error) {
+        
```

Comment:
The previous code used to display an error dialog.
What is the reason to remove it?